examples/datalog/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"github.com/deosjr/whistle/datalog"
+	"github.com/deosjr/whistle/kanren"
+	"github.com/deosjr/whistle/lisp"
+)
+
+func main() {
+	l := lisp.New()
+	kanren.Load(l)
+	datalog.Load(l)
+	l.Eval(`(begin
```

Comment:
each line couldve been wrapped in l.Eval instead of using begin here

Reply:
Understood, but this is less verbose, no?